pinot-query-planner/src/main/java/org/apache/calcite/rel/rules/PinotJoinExchangeNodeInsertRule.java

```diff
@@ -65,6 +65,7 @@ public void onMatch(RelOptRuleCall call) {
 
     if (joinInfo.leftKeys.isEmpty()) {
       // when there's no JOIN key, use broadcast.
+      // TODO: Double check broadcast distribution won't cause problems
```

Comment:
could you explain what kind of problem?

Reply:
I haven't figured out yet. Removed the comment for now